plutarch-test/goldens/api.map.bench.golden

```diff
@@ -21,8 +21,18 @@ difference.emptyRight {"exBudgetCPU":4494995,"exBudgetMemory":15848,"scriptSizeB
 difference.emptyResult {"exBudgetCPU":7452717,"exBudgetMemory":22006,"scriptSizeBytes":179}
 unionWith.const {"exBudgetCPU":9281337,"exBudgetMemory":29270,"scriptSizeBytes":263}
 unionWith.double {"exBudgetCPU":9372814,"exBudgetMemory":28772,"scriptSizeBytes":259}
-unionWith.(+) {"exBudgetCPU":9144040,"exBudgetMemory":28276,"scriptSizeBytes":265}
-unionWith.flip (+) {"exBudgetCPU":9144040,"exBudgetMemory":28276,"scriptSizeBytes":265}
+unionWith.(+) {"exBudgetCPU":9372814,"exBudgetMemory":28772,"scriptSizeBytes":259}
```

Comment:
This is slightly more expensive now.

Reply:
I changed those to to have matching keys in the argument Maps. Before, the combine function wouldn't even be called.